examples/grpc-envoy/bin/main/envoy/launch_envoy.sh

```
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+set -Eeuo pipefail
+
+SED_COMMAND=$1
+
+CONFIG=$(cat $2)
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)